kadasrouting/gui/valhallaroutebottombar.py

```diff
@@ -310,9 +325,12 @@ def clearPins(self):
         """
         # remove origin pin
         self.originSearchBox.removePin()
-        # remove destination poin
+        # remove destination pin
         self.destinationSearchBox.removePin()
 
+    def deletePoints(self):
+        self.originSearchBox.point
```

Comment:
What is this supposed to do?

Reply:
oups should be `self.originSearchBox.point = None` 
good catch, thanks

Reply:
actually no, this was useless